ArthurAtangana/club-management-system
# feat: transition to user dashboards post login

## What type of PR is this? (check all applicable)


## Description
After authenticating a user, system gets the user's group then loads the appropriate dashboard.
Command names are taken from the use case diagrams.

## How to test
* Run the CLI and enter John Smith's credentials to go the admin view (see screenshot).
* Run the CLI and enter Jane Doe's credentials to go to the trainer's view
* Run the CLI and enter Tom Black's credentials to go to the trainer's view

## Added/updated tests?

- [X] No, and this is why: No new interactions with DB.

## Screenshots
<img width="864" alt="image" src="https://github.com/ArthurAtangana/club-management-system/assets/77079888/650b5179-0ad9-4eed-95f3-cfd9b2edd0ad">
